.sqlfluffignore

```diff
@@ -0,0 +1,49 @@
+dist/
+node_modules/
+
+# Migrations are immutable, so already-committed migrations cannot be corrected.
+# https://github.com/zakpatterson/postgres-schema-migrations/blob/75f22a65d3a8ddd16356e74538316fec2ca0202e/README.md#hash-checks-for-previous-migrations
+src/database/migrations/0001-create-canonical_fields.sql
```

Comment:
I was going to say "this approach adds a chore when we write a migration" but then realized "oh, no it doesn't! We don't *want* to ignore new migrations"

w00t!

Reply:
Exactly!

The only exception is if we later change our rules, we'll have to ignore the migrations that passed the old rules. Hopefully that will be sufficiently rare to not need to worry about!